### DISCUSSION TITLE
POS less accurate for PROPN in V3 (fr_core_news_lg) than in V2 with the same model

### DISCUSSION THREAD

**Discussion by bissana** [explosion/spaCy#8380] [Help: Coding & Implementations] [answered]
Hello,
I was using Spacy V2 with the corresponding large pre-trained French model to detect POS, especially proper nouns. It was working GREATE !  For information the processed text is always in lower case.
I switched to V3 and found out that it detects PROPN as nouns. 
Is there a way to get the same POS detection with V3 as for V2 for PROPN at least?

Thank you in advance,
Bissan

**Comment by adrianeboyd** [comment, marked as answer]
There was a bug in the v3.0.x model training related exactly to lowercase texts. The model training should have used the same lowercase augmentation as in v2.3.x, but it was omitted unintentionally, so in the end the v3.0.x models were only trained on texts with newspaper/edited mixed case with capitalized names.

The bugs were fixed in v3.0.4 and the upcoming v3.1.x models should be more like v2.3.x in this respect. (I can't make promises about the release schedule, but within a week or two would be my current guess.)

If you're interested in training your model in the meanwhile (the data is all public), I can give you a few tips about where to start.

**Reply by bissana** [reply]
Thanks a lot for the quick answer !!! 
I would love to get some tips about training the model meanwhile, thanks in advance !

**Reply by adrianeboyd** [reply]
Sure, if you're mainly interested in tagging and parsing, you can use this example project to get started:

https://github.com/explosion/projects/tree/v3/pipelines/tagger_parser_ud

```bash
python -m spacy project clone pipelines/tagger_parser_ud
```

In `project.yml`, switch the `lang` to `fr` , switch `UD_English-EWT` and the related filenames to `UD_French-Sequoia`/`fr_sequoia-ud`

In `configs/default.cfg` add a block like this to the config in place of `augmenter = null` for the training corpus:

```ini
[corpora.train.augmenter]
@augmenters = "spacy.lower_case.v1"
level = 0.1
```

This would be mainly to test to make sure the proper nouns improve as expected and to have a complete working example to refer to.

For some further improvements, you can include vectors (and some minor bugfixes) by using an updated config created with `spacy init config`:

```bash
python -m spacy init config -l fr -p morphologizer,parser -o accuracy config.cfg
```

**Reply by bissana** [reply]
Thank you very much for these helpful tips ! I will surely include vectors for semantic similarity, this is essential in my project.
Have a nice day !
